proxies/live/apiproxy/proxies/default.xml

```diff
@@ -617,8 +639,12 @@
         <Condition>(proxy.pathsuffix MatchesPath "/_ping") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
     </RouteRule>
     <RouteRule name="identityRoute">
-        <Condition>(proxy.pathsuffix MatchesPath "/userinfo") or (proxy.pathsuffix MatchesPath "/.well-known/*")</Condition>
+        <Condition>proxy.pathsuffix MatchesPath "/.well-known/*"</Condition>
```

Comment:
is this actually the behaviour we want? might be quicker to have a chat about this to understand

Reply:
I'm not really sure. This route rule was already there when I started. I don't think it will work anyway because the .well-known path of the external identity server is `/openam/oauth2/realms/root/realms/oidc/.well-known` not `/.well-known`

Reply:
Could just remove this rule?

Reply:
yeah you could. it was a holdover from the original implementation from Apigee.. I don't think it should have been there in the first place.